pom.xml

```diff
@@ -333,26 +333,26 @@
                         <exclude>**/wait-for-it.sh</exclude>
                         <exclude>**/*.out</exclude>
                         <!-- Directory containing dependencies downloaded by NPM -->
-                        <exclude>node_modules/**</exclude>
+                        <exclude>**/node_modules/**</exclude>
                         <!-- Nodejs installed locally by the frontend-maven-plugin -->
-                        <exclude>node/**</exclude>
+                        <exclude>**/node/**</exclude>
                         <!-- Javascript code coverage report generated by Istanbul -->
                         <exclude>coverage/**</exclude>
                         <!-- ACE editor assets are covered in the metron-config NOTICE file -->
                         <exclude>**/src/assets/ace/**</exclude>
-                        <exclude>dist/assets/ace/**</exclude>
+                        <exclude>**/dist/assets/ace/**</exclude>
                         <!-- Generated svg and bundle.css containing Font Awesome fonts are covered in the
                              metron-interface/metron-config README and NOTICE files -->
-                        <exclude>dist/*.svg</exclude>
-                        <exclude>dist/styles.*.bundle.css</exclude>
+                        <exclude>**/dist/*.svg</exclude>
+                        <exclude>**/dist/styles.*.bundle.css</exclude>
                         <!-- 3rdpartylicenses.txt is an empty file carried along by imported libraries -->
-                        <exclude>dist/3rdpartylicenses.txt</exclude>
-                        <exclude>e2e/*.js.map</exclude>
+                        <exclude>**/dist/3rdpartylicenses.txt</exclude>
+                        <exclude>**/e2e/*.js.map</exclude>
                         <!-- Checkstyle is LGPL.  We derive ours from their base, but don't ship it, so it's fine use.
                              Since it is derived, we shouldn't get an Apache header -->
                         <exclude>checkstyle.xml</exclude>
                         <!-- Alerts UI e2e test data -->
-                        <exclude>e2e/mock-data/alerts_ui_e2e_index.data</exclude>
+                        <exclude>**/e2e/mock-data/alerts_ui_e2e_index.data</exclude>
                         <!-- retry files generated by Ansible during a failure -->
```

Comment:
License header check gave false alarms on my local machine without these changes. Now it works both local and Travis.